api/envoy/config/metrics/v3/stats.proto

```diff
@@ -259,6 +262,47 @@ message TagSpecifier {
   }
 }
 
+// Specifies a matcher for stats and the buckets that matching stats should use.
+message HistogramBucketSettings {
+  // The stats that this rule applies to.
+  type.matcher.v3.StringMatcher match = 1 [(validate.rules).message = {required: true}];
+
+  // Each value is the upper bound of a successive bucket.
+  repeated double buckets = 2 [(validate.rules).repeated = {min_items: 2}];
```

Comment:
Why can't you have a single bucket histogram?

Reply:
Yeah, good point. I hadn't thought about that, but there's not a good reason to restrict it further.